SU-SWS/decanter
# Bourbon update

# READY FOR REVIEW

# Summary
- Updated to latest version of Bourbon (v5.0.0)
- Updated all deprecated code that we were utilizing (namely `font-face` and Font stack variables)
- Latest version removes prefixing features, so have added autoprefixer and postcss (plus related Grunt command) to add vendor prefixes only where needed (https://github.com/postcss/autoprefixer); also added postcss as part of watch task

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that local version of style guide and pattern library has remained unchanged (visually)
3. Can also test new grunt command (`grunt postcss`); also runs with `grunt watch` if running when change is made

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/issues/112
- https://www.bourbon.io/docs/migrating-from-v4-to-v5/

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
